recipes/demes/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "demes" %}
+{% set version = "0.1.0" %}
+{% set sha256 = "40b515f5ad2469a7de1a2a4c0eb91b8ad5a7a39e9038c3b50d64a909face5b4a" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  noarch: python
+
+requirements:
+  host:
+    - python >=3.6
+    - setuptools
+    - setuptools_scm
+    - pip
+
+  run:
+    - python >=3.6
+    # pins taken from setup.cfg
+    - attrs  >=20.3
+    - ruamel.yaml >=0.15.78
+
+test:
+  imports:
+    - demes
+  commands:
+    - demes --help
```

Comment:
I don't think this will work. Probably we should fix that (the CLI isn't documented anyhow).

Reply:
Right - will I just remove this then? We probably shouldn't be installing a ``demes`` program into the users PATH if it's not supported, though.

Reply:
You're right. To be honest, I just forgot about it.